test/test_boto_utils.py

```diff
@@ -99,4 +99,4 @@ def test_md5sum_with_size(self):
             key.set_contents_from_string("test")
             (md5, size) = md5sum_with_size(key)
             assert size == key.size
-            assert md5 == key.md5
+            assert md5.encode('ascii') == key.md5
```

Comment:
Huh, I didn't realize Boto exposed the md5 as bytes. :| I guess it never came up in Python 2.